vm/devices/net/netvsp/src/lib.rs

```diff
@@ -4929,6 +4982,13 @@ impl<T: 'static + RingMem> NetChannel<T> {
         data: &mut ProcessingData,
         epqueue: &mut dyn net_backend::Queue,
     ) -> Result<bool, WorkerError> {
+        if self.packet_filter == rndisprot::NDIS_PACKET_TYPE_NONE {
+            tracing::debug!(
```

Comment:
I'd suggest `trace` level.

Reply:
(`debug` is enabled in automated testing, and it seems like this could get quite noisy.)